deployment/CP04OSSM_Deploy.csv

```diff
@@ -309,65 +309,65 @@ AR34,,AR39,,CP04OSSM-SBD11-06-METBKA000,10,1,2019-04-05T14:55:00,2019-09-28T12:3
+AR48,,,,CP04OSSM-MFD37-03-CTDBPE000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-CTDBPE-50004,39.939883,-70.876693,,442,443,
+AR48,,,,CP04OSSM-MFD37-04-DOSTAD000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-DOSTAD-00392,39.939883,-70.876693,,442,443,
+AR48,,,,CP04OSSM-MFD37-07-ZPLSCC000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-ZPLSCC-55077,39.939883,-70.876693,,442,443,
+AR48,,,,CP04OSSM-RIC21-00-CPMENG000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGCON-NCPM02-00009,39.939883,-70.876693,,7,443,
+AR48,,,,CP04OSSM-RID26-00-DCLENG000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGCON-NDCL26-00015,39.939883,-70.876693,,7,443,
+AR48,,,,CP04OSSM-RID26-04-VELPTA000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-VELPTA-12229,39.939883,-70.876693,,7,443,
+AR48,,,,CP04OSSM-RID26-06-PHSEND000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-PHSEND-P0099,39.939883,-70.876693,,7,443,
+AR48,,,,CP04OSSM-RID26-07-NUTNRB000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-NUTNRB-01107,39.939883,-70.876693,,7,443,
+AR48,,,,CP04OSSM-RID26-08-SPKIRB000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-SPKIRB-00289,39.939883,-70.876693,,7,443,
+AR48,,,,CP04OSSM-RID27-00-DCLENG000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGCON-NDCL27-00009,39.939883,-70.876693,,7,443,
+AR48,,,,CP04OSSM-RID27-01-OPTAAD000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-OPTAAD-00206,39.939883,-70.876693,,7,443,
+AR48,,,,CP04OSSM-RID27-02-FLORTD000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-FLORTD-01216,39.939883,-70.876693,,7,443,
+AR48,,,,CP04OSSM-RID27-03-CTDBPC000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-CTDBPC-50003,39.939883,-70.876693,,7,443,
+AR48,,,,CP04OSSM-RID27-04-DOSTAD000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-DOSTAD-00391,39.939883,-70.876693,,7,443,
+AR48,,,,CP04OSSM-SBC11-00-CPMENG000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGCON-BCPM01-50002,39.939883,-70.876693,,0,443,
+AR48,,,,CP04OSSM-SBD11-00-DCLENG000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGCON-BDCL11-50002,39.939883,-70.876693,,0,443,
+AR48,,,,CP04OSSM-SBD11-01-MOPAK0000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-MOPAK0-44005,39.939883,-70.876693,,0,443,
+AR48,,,,CP04OSSM-SBD11-02-HYDGN0000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-HYDGN0-46156,39.939883,-70.876693,,0,443,
+AR48,,,,CP04OSSM-SBD11-06-METBKA000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-METLGR-00038,39.939883,-70.876693,,0,443,
+AR48,,,,CP04OSSM-SBD12-00-DCLENG000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGCON-BDCL12-50002,39.939883,-70.876693,,0,443,
+AR48,,,,CP04OSSM-SBD12-03-HYDGN0000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-HYDGN0-41552,39.939883,-70.876693,,0,443,
+AR48,,,,CP04OSSM-SBD12-04-PCO2AA000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-PCO2AA-35260,39.939883,-70.876693,,0,443,
```

Comment:
Looks good but check the FLORT S/N swapout (I think its correct).

Reply:
Looks good. Yes FLORT 01214 was on ISSM.  FLORT 01216 is correct here.